quodlibet/quodlibet/qltk/renamefiles.py

```diff
@@ -282,6 +336,77 @@ def __rename(self, library):
         library.changed(was_changed)
         self.save.set_sensitive(False)
 
+    def __moveart(self, art_sets, pathfile_old, pathfile_new, song):
+
+        path_old = os.path.dirname(os.path.realpath(pathfile_old))
+        path_new = os.path.dirname(os.path.realpath(pathfile_new))
+        if os.path.samefile(path_old, path_new):
+            return
+        if (path_old in art_sets.keys() and len(art_sets[path_old]) == 0):
+            return
+
+        # get art set for path
+        images = []
+        if path_old in art_sets.keys():
+            images = art_sets[path_old]
+        else:
+            def glob_escape(s):
+                for c in ['[', '*', '?']:
```

Comment:
BTW neater is probably `for c in '[*?':`

Reply:
hehe, er, yes.. :)